src/engine/openspaceengine.cpp

```diff
@@ -497,22 +499,124 @@ void OpenSpaceEngine::initialize() {
     // Load a light and a monospaced font
     loadFonts();
 
-    // Initialize the Scene
-    // @CLEANUP:  This should become a unique_ptr that is either created inside the
-    // renderengine or moved into it ---abock
-    Scene* sceneGraph = new Scene;
-    sceneGraph->initialize();
-    
     std::string scenePath = "";
     configurationManager().getValue(ConfigurationManager::KeyConfigScene, scenePath);
-    sceneGraph->scheduleLoadSceneFile(scenePath);
 
-    // Initialize the RenderEngine
-    _renderEngine->setSceneGraph(sceneGraph);
     _renderEngine->initialize();
+
+    scheduleLoadScene(scenePath);
+
+    LINFO("Finished initializing");
+}
+
+void OpenSpaceEngine::scheduleLoadScene(const std::string& scenePath) {
```

Comment:
Here you can pass `scenePath` as a `std::string` instead and `std::move` it into the `_scenePath` variable.  You'll potentially save one copy